modules/model/shared/src/main/scala/observe/model/ModelLenses.scala

```diff
@@ -38,10 +36,10 @@ trait ModelLenses {
 //  val parametersRoot: Iso[Map[ParamName, ParamValue], Map[ParamName, ParamValue]]   =
 //    Iso.id[Map[ParamName, ParamValue]]
 
-  val sequenceStepT: Traversal[SequenceView, Step] =
```

Comment:
It seems we will no longer have sequences in the shared or even the server model. I was thinking of removing these `Traversal`s.

Reply:
👍